rocAL_pybind/rocal_pybind.cpp

```diff
@@ -448,8 +456,6 @@ PYBIND11_MODULE(rocal_pybind, m) {
         .export_values();
     py::class_<ROIxywh>(m, "ROIxywh")
         .def(py::init<>())
```

Comment:
Is this used? Why is this removed?

Reply:
added back. Should have been removed unintenionally